pom.xml

```diff
@@ -1091,12 +1091,6 @@
 		-->
 		<profile>
 			<id>windows</id>
-			<activation>
-				<activeByDefault>false</activeByDefault>
-				<os>
-					<family>windows</family>
-				</os>
-			</activation>
```

Comment:
@SubJunk this deletion stopped making the "UMS-setup.exe". Do not ask me why. I realized it when I deleted the Eclipse "target" folder and tried to make distributable files.

Reply:
Which command do you use? I do `mvn -P windows package`